packages/drivers/odsp-driver/src/odspCache.ts

```diff
@@ -219,6 +233,16 @@ export class LocalPersistentCache implements IPersistedCache {
     updateUsage(entry: ICacheEntry, seqNumber: number): void {
     }
 
+    removeAllEntriesForDocId(docId: string): void {
+        Array.from(this.cache)
+        .filter(([key]) => {
+            key.startsWith(docId);
+        })
+        .map(([key]) => {
+            this.cache.delete(key);
+        });
```

Comment:
Not sure how big this cache would get for a particular docId, but would it be better to just do one pass over the cache like this, instead of two (`.filter` and `.map`)?
```
        .map(([key]) => {
            if (key.startsWith(docId) {
                this.cache.delete(key);
            }
        });
```